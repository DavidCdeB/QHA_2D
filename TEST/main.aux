\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\beamer@endinputifotherversion {3.36pt}}
\@writefile{nav}{\beamer@endinputifotherversion {3.36pt}}
\select@language{english}
\@writefile{toc}{\select@language{english}}
\@writefile{lof}{\select@language{english}}
\@writefile{lot}{\select@language{english}}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {1}{1}}}
\@writefile{nav}{\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {2}{2}}}
\@writefile{toc}{\beamer@sectionintoc {1}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{3}{0}{1}}
\@writefile{nav}{\headcommand {\sectionentry {1}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{3}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{0}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {1}{2}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {1}{2}}}
\newlabel{sec:1}{{1}{3}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {sec:1}{3}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {3}{3}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{2}{4/4}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {4}{4}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{3}{5/5}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {5}{5}}}
\@writefile{nav}{\headcommand {\slideentry {1}{0}{4}{6/23}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {6}{23}}}
\@writefile{toc}{\beamer@sectionintoc {2}{Shapes of $G$, $F$, $\mathcal  {E}$}{24}{0}{2}}
\@writefile{nav}{\headcommand {\sectionentry {2}{Shapes of $G$, $F$, $\mathcal  {E}$}{24}{Shapes of $G$, $F$, $\mathcal  {E}$}{0}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {3}{23}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {3}{23}}}
\newlabel{sec:2}{{2}{24}{Shapes of $G$, $F$, $\mathcal {E}$}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {sec:2}{24}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{1}{24/24}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {24}{24}}}
\@writefile{nav}{\headcommand {\slideentry {2}{0}{2}{25/25}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {25}{25}}}
\@writefile{toc}{\beamer@sectionintoc {3}{Shape of $H(P;T)$}{26}{0}{3}}
\@writefile{nav}{\headcommand {\sectionentry {3}{Shape of $H(P;T)$}{26}{Shape of $H(P;T)$}{0}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {24}{25}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {24}{25}}}
\newlabel{sec:3}{{3}{26}{Shape of $H(P;T)$}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {sec:3}{26}}
\@writefile{nav}{\headcommand {\slideentry {3}{0}{1}{26/26}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {26}{26}}}
\@writefile{nav}{\headcommand {\slideentry {3}{0}{2}{27/27}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {27}{27}}}
\@writefile{toc}{\beamer@sectionintoc {4}{Phase Boundary}{28}{0}{4}}
\@writefile{nav}{\headcommand {\sectionentry {4}{Phase Boundary}{28}{Phase Boundary}{0}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {26}{27}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {26}{27}}}
\newlabel{sec:4}{{4}{28}{Phase Boundary}{Doc-Start}{}}
\@writefile{snm}{\beamer@slide {sec:4}{28}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{1}{28/28}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {28}{28}}}
\@writefile{nav}{\headcommand {\slideentry {4}{0}{2}{29/29}{}{0}}}
\@writefile{nav}{\headcommand {\beamer@framepages {29}{29}}}
\@writefile{nav}{\headcommand {\beamer@partpages {1}{29}}}
\@writefile{nav}{\headcommand {\beamer@subsectionpages {28}{29}}}
\@writefile{nav}{\headcommand {\beamer@sectionpages {28}{29}}}
\@writefile{nav}{\headcommand {\beamer@documentpages {29}}}
\@writefile{nav}{\headcommand {\def \inserttotalframenumber {12}}}
\expandafter\ifx\csname c@section@totc\endcsname\relax\newcounter{section@totc}\fi\setcounter{section@totc}{4}
