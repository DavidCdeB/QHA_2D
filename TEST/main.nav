\beamer@endinputifotherversion {3.36pt}
\headcommand {\slideentry {0}{0}{1}{1/1}{}{0}}
\headcommand {\beamer@framepages {1}{1}}
\headcommand {\slideentry {0}{0}{2}{2/2}{}{0}}
\headcommand {\beamer@framepages {2}{2}}
\headcommand {\sectionentry {1}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{3}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{0}}
\headcommand {\beamer@sectionpages {1}{2}}
\headcommand {\beamer@subsectionpages {1}{2}}
\headcommand {\slideentry {1}{0}{1}{3/3}{}{0}}
\headcommand {\beamer@framepages {3}{3}}
\headcommand {\slideentry {1}{0}{2}{4/4}{}{0}}
\headcommand {\beamer@framepages {4}{4}}
\headcommand {\slideentry {1}{0}{3}{5/5}{}{0}}
\headcommand {\beamer@framepages {5}{5}}
\headcommand {\slideentry {1}{0}{4}{6/23}{}{0}}
\headcommand {\beamer@framepages {6}{23}}
\headcommand {\sectionentry {2}{Shapes of $G$, $F$, $\mathcal {E}$}{24}{Shapes of $G$, $F$, $\mathcal {E}$}{0}}
\headcommand {\beamer@sectionpages {3}{23}}
\headcommand {\beamer@subsectionpages {3}{23}}
\headcommand {\slideentry {2}{0}{1}{24/24}{}{0}}
\headcommand {\beamer@framepages {24}{24}}
\headcommand {\slideentry {2}{0}{2}{25/25}{}{0}}
\headcommand {\beamer@framepages {25}{25}}
\headcommand {\sectionentry {3}{Shape of $H(P;T)$}{26}{Shape of $H(P;T)$}{0}}
\headcommand {\beamer@sectionpages {24}{25}}
\headcommand {\beamer@subsectionpages {24}{25}}
\headcommand {\slideentry {3}{0}{1}{26/26}{}{0}}
\headcommand {\beamer@framepages {26}{26}}
\headcommand {\slideentry {3}{0}{2}{27/27}{}{0}}
\headcommand {\beamer@framepages {27}{27}}
\headcommand {\slideentry {3}{0}{3}{28/28}{}{0}}
\headcommand {\beamer@framepages {28}{28}}
\headcommand {\beamer@partpages {1}{28}}
\headcommand {\beamer@subsectionpages {26}{28}}
\headcommand {\beamer@sectionpages {26}{28}}
\headcommand {\beamer@documentpages {28}}
\headcommand {\def \inserttotalframenumber {11}}
