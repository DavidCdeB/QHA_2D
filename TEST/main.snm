\beamer@slide {sec:1}{3}
\beamer@slide {sec:2}{24}
\beamer@slide {sec:3}{26}
