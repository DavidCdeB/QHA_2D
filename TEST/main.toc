\beamer@endinputifotherversion {3.36pt}
\select@language {english}
\beamer@sectionintoc {1}{$F(V)$, $G(P)$ and pressure of intersection. Thermal evolution}{3}{0}{1}
\beamer@sectionintoc {2}{Shapes of $G$, $F$, $\mathcal {E}$}{24}{0}{2}
\beamer@sectionintoc {3}{Shape of $H(P;T)$}{26}{0}{3}
