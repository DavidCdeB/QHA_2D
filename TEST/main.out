\BOOKMARK [2][]{Outline0.1}{F\(V\), G\(P\) and pressure of intersection. Thermal evolution}{}% 1
\BOOKMARK [2][]{Outline0.2}{Shapes of G, F, E}{}% 2
\BOOKMARK [2][]{Outline0.3}{Shape of H\(P;T\)}{}% 3
\BOOKMARK [2][]{Outline0.4}{Phase Boundary}{}% 4
